dataprofiler/profilers/data_labeler_column_profile.py

```diff
@@ -27,15 +27,20 @@ def __init__(self, name, data_labeler_dirpath=None, options=None):
             if not isinstance(options, DataLabelerOptions):
                 raise ValueError("DataLabelerColumn parameter 'options' must be"
                                  " of type DataLabelerOptions.")
-            if options.data_labeler_dirpath:
-                data_labeler_dirpath = options.data_labeler_dirpath
             if options.max_sample_size:
                 self._max_sample_size = options.max_sample_size
 
-        self.data_labeler = DataLabeler(
-            labeler_type='structured',
-            dirpath=data_labeler_dirpath,
-            load_options=None)
+        if data_labeler is None:
+            data_labeler_dirpath = None
+            if options and options.data_labeler_dirpath:
+                data_labeler_dirpath = options.data_labeler_dirpath
+
+            data_labeler = DataLabeler(
+                labeler_type='structured',
+                dirpath=data_labeler_dirpath,
+                load_options=None)
+
+        self.data_labeler = data_labeler
```

Comment:
This is super weird.

Reply:
what are multiple ways? Can you give an example?

Reply:
We need to throw a warning if there's a `data_labeler_dirpath` which differs from the `data_labeler`. `data_labeler` will also need to keep track of its source directory. Tests will need to be written.